src/frontend/src/components/pinInput.ts

```diff
@@ -1,4 +1,5 @@
 import { toast } from "$src/components/toast";
+import { DynamicKey } from "$src/utils/i18n";
```

Comment:
don't know if you use this in II but, `import type { DynamicKey } from "$src/utils/i18n";` would work here

Reply:
It is, thanks!